src/main/kotlin/racingcar/model/car/Car.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.model.car
+
+open class Car(name: String, _position: Int = 0) {
+    var carName: CarName = CarName(name)
+        private set
```

Comment:
```suggestion
    val carName: CarName = CarName(name)
```
그냥 불변으로 선언 하셔도 좋을거 같아요!

Reply:
헉 이런 실수를..!
캐치해주셔서 감사합니다!